composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "kabbouchi/laravel-logger-discord-channel",
+  "name": "adlerdias/laravel-logger-discord-channel",
```

Comment:
can u revert this change?:)

Reply:
Of course I can, I'm really sorry about that.